releases/release-5.4.0.md

```diff
@@ -0,0 +1,435 @@
+    - More components supported: Besides TiDB, PD, and TiKV, TiDB v5.4.0 also supports CPU profiling of TiFlash.
+    - More forms of profiling display: Supports showing CPU profiling and Goroutine results on flame charts.
+
+    - More deployment environments supported: Continuous Profiling can also be used for clusters deployed using TiDB Operator.
+
+   Continuous Profiling is disabled by default and can be enabled on TiDB Dashboard. 
+
+    Continuous Profiling is applicable to clusters deployed or upgraded using TiUP of v1.9.0 or later or TiDB Operator of v1.3.0 or later.
+
+    [User document](/dashboard/continuous-profiling.md)
+
+## Improvements
+
++ TiDB
+
+    - Add a new system variable `tidb_enable_paging` to determine whether to use paging to send Coprocessor requests. Enabling this feature can reduce the amount of data to process and to reduce latency and resource consumption [#30578](https://github.com/pingcap/tidb/issues/30578)
+    - Support the `ADMIN {SESSION | INSTANCE | GLOBAL} PLAN_CACHE` syntax to clear the cached query plan [#30370](https://github.com/pingcap/tidb/pull/30370)
+
++ TiKV
+
+    - Coprocessor supports paging API to process requests in a stream-like way [#11448](https://github.com/tikv/tikv/issues/11448)
+    - Support `read-through-lock` so that read operations do not need to wait for secondary locks to be resolved [#11402](https://github.com/tikv/tikv/issues/11402)
+    - Add a disk protection mechanism to avoid panic caused by disk space drainage [#10537](https://github.com/tikv/tikv/issues/10537)
+    - Support archiving and rotating logs [#11651](https://github.com/tikv/tikv/issues/11651)
+    - Reduce the system call by the Raft client and increase CPU efficiency [#11309](https://github.com/tikv/tikv/issues/11309)
+    - Coprocessor supports pushing down substring to TiKV [#11495](https://github.com/tikv/tikv/issues/11495)
+    - Improve the scan performance by skip reading locks in the Read Committed isolation level [#11485](https://github.com/tikv/tikv/issues/11485)
+    - Reduce the default thread pool size used by backup operations and limit the use of thread pool when the stress is high [#11000](https://github.com/tikv/tikv/issues/11000)
+    - Support dynamically adjusting the sizes of the Apply thread pool and the Store thread pool [#11159](https://github.com/tikv/tikv/issues/11159)
+    - Support configuring the size of the `snap-generator` thread pool [#11247](https://github.com/tikv/tikv/issues/11247)
+    - Optimize the issue of global lock race that occurs when there are many files with frequent reads and writes [#250](https://github.com/tikv/rocksdb/pull/250)
+
++ PD
+
+    - Record the historic hotspot information by default [#25281](https://github.com/pingcap/tidb/issues/25281)
+    - Add signature for the HTTP component to identify the request source [#4490](https://github.com/tikv/pd/issues/4490)
+    - Update TiDB Dashboard to v2021.12.31 [#4257](https://github.com/tikv/pd/issues/4257)
```

Comment:
Can we link to release notes or otherwise list the major improvements since the last version?